Userland/Libraries/LibWeb/Layout/BlockFormattingContext.cpp

```diff
@@ -135,11 +135,6 @@ bool BlockFormattingContext::box_should_avoid_floats_because_it_establishes_fc(B
 
 void BlockFormattingContext::compute_width(Box const& box, AvailableSpace const& available_space)
 {
-    if (box.is_absolutely_positioned()) {
```

Comment:
Should we try to make this a bit more verbose and add a verify?

```cpp
void BlockFormattingContext::compute_width_for_block_level_box(Box const& box, AvailableSpace const& available_space)
{
    VERIFY(!box.is_absolutely_positioned());
```

Reply:
we can, but don't you think having this check only in `compute_width()` is a bit out of place considering all other methods simply assume they are never invoked for abspos because `layout_block_level_box` filters them out?

Reply:
Yeah yeah, fine :)